gf-guide/description.md

```diff
@@ -108,3 +108,19 @@ while Katakana and Kanji are slightly heavier, which gives the unique rhythm and
 > *Zen Antique* には、古風な雰囲気の二種類の漢字を含む日本語書体があります。 文字によって画線の太さに変化があり、ひらがなと欧文は細め、カタカナと漢字は太めで、フォントに独特のリズムと味わいを与えています。 また、*[Zen Antique Soft](https://fonts.google.com/specimen/Zen+Antique+Soft?subset=japanese)* では、角が少し丸くなっています。
 >
 > このプロジェクトに参加して貢献したい方は、次の URL をご参照ください。*[github.com/googlefonts/zen-antique](https://github.com/googlefonts/zen-antique)*
+
+## The Article
+
+The Article is a new expanded version of the `DESCRIPTION.en_us.html` file that can include images and longer-form information and content. It appears on the Google Fonts site as the main content in the About & License section. For example, see the About page for [Gulzar](https://fonts.google.com/specimen/Gulzar/about), and then view the `ofl/gulzar` directory in the Google Fonts repo: [github.com/google/fonts/tree/main/ofl/gulzar](https://github.com/google/fonts/tree/main/ofl/gulzar). 
+
+The `ARTICLE.en_us.html` file and images should live in an `article/` directory alongside the `METDATA.pb` file in the root of the `ofl/fontname` directory. It is expected that only the Noto fonts combine the `DESCRIPTION.en_us.html` and `ARTICLE.en_us.html` files. All the other families (non-Noto) will have their descriptions overwritten by Article. This means we can shift all the Description content into the Article content and delete the `DESCRIPTION.en_us.html` from the font directory.
```

Comment:
> will have their descriptions overwritten by Article.

I don't think this will be the case for all the families. Some will continue to be onboarded without images, so not sure if the Description will continue to work for this purpose.